app/javascript/mastodon/locales/defaultMessages.json

```diff
@@ -319,6 +319,15 @@
     ],
     "path": "app/javascript/mastodon/containers/status_container.json"
   },
+  {
+    "descriptors": [
+      {
+        "defaultMessage": "Media",
+        "id": "account.media"
+      }
+    ],
+    "path": "app/javascript/mastodon/features/account_gallery/index.json"
+  },
```

Comment:
[This message has been deleted](https://github.com/tootsuite/mastodon/commit/51310125051a75ef7af4e8ffc8b6532c151e96b6#diff-78619937f6391fab9cb5ecbef39df761L93)